src/pages/Dashboard/Automations/AddAction.vue

```diff
@@ -130,6 +130,13 @@ export default {
     },
     disableNext() {
       if (this.step.name === 'openToConfig') {
+        if (
+          this.isPagerDuty &&
+          this.apiToken &&
+          this.routingKey &&
+          this.severity
+        )
+          return false
```

Comment:
Do we need to add these checks to the other action types, similar to https://github.com/PrefectHQ/ui/pull/835/files#diff-3af46aeb97e3ab5dfecf9ff91442aa60a6bfd556c5f1b40ea61a760ffdfc3334L141-L158?